sql/catalyst/src/main/scala/org/apache/spark/sql/types/StructType.scala

```diff
@@ -453,7 +453,23 @@ case class StructType(fields: Array[StructField]) extends DataType with Seq[Stru
    * 3. If B doesn't exist in `this`, it's also included in the result schema.
    * 4. Otherwise, `this` and `that` are considered as conflicting schemas and an exception would be
    *    thrown.
+   *
```

Comment:
The statements below are duplicate with ones in L447-L455. Can you add some statements for this fix based on the existing  L447-L455 statements instead of totally re-writing comments?

Reply:
Okay, I've added a statement to the existing comment